packages/main/src/plugin/mcp/mcp-registry.ts

```diff
@@ -374,6 +416,35 @@ export class MCPRegistry {
     });
   }
 
+  protected async setupPackage(
+    pack: components['schemas']['Package'] | undefined,
+    config: PackageStorageConfigFormat,
+  ): Promise<Transport> {
+    if (!pack) throw new Error('package not found');
+
+    const resolved: components['schemas']['Package'] = {
+      ...pack,
+      runtime_arguments: (pack.runtime_arguments ?? []).map((argument, index) => ({
+        ...argument,
+        value: config.runtimeArguments[index],
+        variables: undefined,
+      })),
+      package_arguments: (pack.package_arguments ?? []).map((argument, index) => ({
+        ...argument,
+        value: config.packageArguments[index],
+        variables: undefined,
+      })),
+      environment_variables: (pack.environment_variables ?? []).map(argument => ({
+        ...argument,
+        value: config.environmentVariables[argument.name],
+        variables: undefined,
+      })),
+    };
```

Comment:
I've noticed that now they changed to camelCase... 😢
https://github.com/modelcontextprotocol/registry/pull/522

Reply:
Was a bit annoying but thanks to typescript, fixed all the problem :+1: